app/src/main/java/org/cpen321/discovr/MainActivity.java

```diff
@@ -114,6 +127,7 @@ protected void onCreate(Bundle savedInstanceState) {// Get the SearchView and se
         mapFragment.getMapAsync(new OnMapReadyCallback() {
             @Override
             public void onMapReady(MapboxMap mapboxMap) {
+                map = mapboxMap;
```

Comment:
I would not recommend doing this because OnMapReady guarantees that the map is in a stable state; if the map somehow goes invalid and we call any action on the saved map it will most likely do something bad.